### DISCUSSION TITLE
IKEA Bulb: State is off but light stays on

### DISCUSSION THREAD

**Discussion by BiasF** [Koenkk/zigbee2mqtt#9979] [Devices]
Hi,

I have some issues with IKEA Bulbs. Sometimes if I turn them off these bulbs, state in z2m switches to "off" but the light stays on. If this occures, I have to turn the light on and off again then the light goes out.

Has somone an idea what the problem is?

Tobias

**Comment by dumbo0001** [comment]
I have the same. It occurs often and today I finally checked the zigbee2mqtt logs when I expect it to happen. I don't have the exact log message because I was on my mobile. I saw that no error or warn message but a normal INFO message with the state to Off.

I am also curious about the cause of this problem and how we can solve it.
